src/main/java/com/rpg2014/ServerController.java

```diff
@@ -32,8 +37,8 @@
     @Produces(MediaType.APPLICATION_JSON)
     @RequiresLogin
     public StatusResponse serverStatus() {
-//        Status status = EC2Wrapper.getServerStatus();
-        return StatusResponse.builder().status(Status.Terminated).build();
+        Status status = (Status) ec2Invoker.invoke(Ec2MethodNames.Status).get();
```

Comment:
Is this cast safe? I'm not sure of the documentation on the .get() method, or even where it's coming from lol.

Reply:
We throw an error if anything goes badly in the ec2wrapper, so this optional will always return something, and that will be of the return type of the method given to the invoker.  I'm looking into a way to make it typesafe, but idk if we need to invest the effort.